### PR TITLE
Nextjs project cleanup

### LINKED ISSUE #2: Clean Up Basic Next.js Application
The objective of this task is to conduct a comprehensive cleanup of our Next.js application to eliminate any unused content, CSS rules, assets, and other unnecessary elements. This cleanup initiative is intended to enhance the maintainability of the codebase, reduce file size, and ensure that we begin from a clean slate.

## Tasks
1. Identify Unused Content
2. Remove Unused Content
3. Identify and Eliminate Unused elements

## Additional Notes
* Create a dedicated branch for this cleanup task to isolate changes from the main codebase.
* Document significant changes made during the cleanup process through through comments or commit messages.
* Ensure that the cleanup process does not inadvertently remove critical functionality or disrupt existing features.
* Upon completion of the cleanup and successful testing, initiate a pull request for review and merge it into the main branch after receiving approval.

## End Goal
The application should render a blank page with just a (unstyled) title

### PR DESCRIPTION
**Pull request by hardRockWorld**
@ThomasCode92:
I have tried to remove most of the unnecessary files from the project and cleaned up as per our requirements!

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,113 +1,3 @@
...
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50 text-balance`}>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
-    </main>
-  );
+  return <></>;
```
Comment (ThomasCode92) [inline app/page.tsx]: Maybe add some title, so we can see something on the screen and test some things later.

**Review by ThomasCode92 — COMMENTED** [review]
You can also tidy up the **README.md** file. Simply delete all existing content and provide a brief description explaining the purpose of this project.

File: app/page.tsx (hunk 1 above)
Comment (hardRockWorld) [reply app/page.tsx]: It works when I add a paragraph and write Hello World!.. I thought it should be totally blank, that's why I kept it blank

Comment (hardRockWorld) [reply app/page.tsx]: Ok, I will modify the readme file

**Review by ThomasCode92 — APPROVED** [review]

**Pull request merged**